controller/server/main.go

```diff
@@ -43,6 +43,7 @@ var (
 	defaultMetallbManifestDir = ""
 	defaultMeshnetManifestDir = ""
 	defaultIxiaTGManifestDir  = ""
+	defaultSRLinuxManifestDir = ""
```

Comment:
does it make sense to default to srl-controller https uri path as a default for this var?

Reply:
looks like controller/server is something used internally, hence the paths are relative to homedir

Reply:
Yep that's the case